### PR TITLE
Spanish and Japanese translations for reports

### PR DESCRIPTION
**Pull request by cdibella**
## Description

Most of the out-of-the-box reports did not have Spanish or Japanese locales/translations. I added these and made a few other small locale-related fixes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ x] I have read the **CONTRIBUTING** document.
- [ x] I have authority to submit this code.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: reports/accessions/accession_classifications_subreport/es.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+en:
```
Comment (lorawoodford) [inline reports/accessions/accession_classifications_subreport/es.yml:1]:
```suggestion
es:
```

File: reports/accessions/accession_classifications_subreport/ja.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+en:
```
Comment (lorawoodford) [inline reports/accessions/accession_classifications_subreport/ja.yml:1]:
```suggestion
ja:
```

File: reports/accessions/accession_deaccessions_subreport/es.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+en:
```
Comment (lorawoodford) [inline reports/accessions/accession_deaccessions_subreport/es.yml:1]:
```suggestion
es:
```

File: reports/resources/resource_locations_list_report/es.yml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+en:
```
Comment (lorawoodford) [inline reports/resources/resource_locations_list_report/es.yml:1]:
```suggestion
es:
```

**Review by lorawoodford — CHANGES_REQUESTED** [review]
Heya @cdibella - There are just 4 places where the first line is identifying English, instead of Spanish/Japanese.  After those are fixed these look good to go!

**Comment by cdibella** [comment]
Ignore that mess I just put in that commit! Actual fix for this pending.

**Review by lorawoodford — APPROVED** [review]

**Pull request merged**